src/test/java/com/google/javascript/gents/singleTests/interface.ts

```diff
@@ -23,9 +23,8 @@ class Y implements Interface2, Interface3 {
 }
 
 export interface StructuralInterface { bar(a: string): number; }
-
-/** !ns.StructuralInterface */
 const structInterfaceImpl: StructuralInterface = {
+  /** !ns.StructuralInterface */
```

Comment:
Is this an intentional change? It'd be nice if we could keep the comments on the variables. Maybe we can rather change the way how we find the comment instead of changing the way how we link the comments?